vimeo-networking/src/main/java/com/vimeo/networking/model/VimeoAccount.java

```diff
@@ -39,53 +40,65 @@
 public class VimeoAccount implements Serializable {
 
     private static final long serialVersionUID = -8341071767843490585L;
-    //    private static final String TOKEN_TYPE_BEARER = "bearer";
 
+    @Nullable
     @GsonAdapterKey("access_token")
     public String accessToken;
+
+    @Nullable
     @GsonAdapterKey("token_type")
     public String tokenType;
+
+    @Nullable
     @GsonAdapterKey("scope")
     public String scope;
+
+    @Nullable
     @GsonAdapterKey("user")
     public User user;
     private String userJSON;
 
-    public VimeoAccount(){
+    public VimeoAccount() {
         //constructor for stag TypeAdapter generation
     }
 
     public VimeoAccount(@Nullable String accessToken) {
         this.accessToken = accessToken;
```

Comment:
Is there a reason why this class uses this. for its member fields? It's not our general coding style elsewhere. Ditto for the "m" prefix for members.